public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_monitor/azure_monitor_datasource.ts

```diff
@@ -152,83 +157,31 @@ export default class AzureMonitorDatasource extends DataSourceWithBackend<AzureM
     });
   }
 
-  getMetricDefinitions(subscriptionId: string, resourceGroup: string) {
```

Comment:
This method is no longer used since it was only used for template variables, that now uses `getMetricNamespaces`